### DISCUSSION TITLE
GitSavvy + ActualVim

### DISCUSSION THREAD

**Discussion by jaanauati** [timbrel/GitSavvy#1526] [General]
Hi, looks like GitSavvy does not play very well with ActualVim, at my end I have to manually disable ActualVim each time I open GitSavvy tabs (status, diff, etc). 

Do you know of any configuration change/workaround to deal with this?

Thanks.

**Comment by kaste** [comment]
I don't know `ActualVim`, but at least you must set 

```
    /*
        When set to `true`, GitSavvy will become Vintagoues friendly.
        You might need to restart Sublime in order to get this working.
     */
    "vintageous_friendly": false,

    /*
        When set to `true`, all views will start in INSERT mode.
        Ignored if `vintageous_friendly` is `false`.
     */
    "vintageous_enter_insert_mode": false,
```

in the settings.  

If you do so we do run

```
                self.view.run_command("_enter_insert_mode")
                self.view.run_command("nv_enter_insert_mode")  # since NeoVintageous 1.22.0
```

and set

```
	        view.settings().set("__vi_external_disable", False)
```

for each of our views.  I don't know if `ActualVim` supports this but it is how we support vim plugins.  We probably could call other commands here.

**Reply by jaanauati** [reply]
I'll give it a try, THANK YOU!
